src/app/find-help/accommodation-search/accommodation-search.component.html

```diff
@@ -1,4 +1,4 @@
-<mat-card>
+<mat-card id="results_start">
```

Comment:
Don't use IDs in components, you can get duplicates when not expected.

But you won't need it anyway when you apply my other comments.

Reply:
Thanks for the tip, done.